org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/format/DOMAttributeFormatter.java

```diff
@@ -75,6 +84,10 @@ public void formatAttribute(DOMAttr attr, int prevOffset, boolean singleAttribut
 				// 2.2 Remove extra spaces between delimiter and start of attribute value
 				int attrValueStart = attr.getNodeAttrValue().getStart(); // <foo attr = |""
 				removeLeftSpaces(delimiterOffset, attrValueStart, edits);
+				// 2.3 Remove extra spaces between start of attribute value quote and actual value
+				//removeLeftSpaces(attrValueStart + 1, // <foo attr = "|"

```

Comment:
This removes the space between the first quote and the start of the actual attribute value. This is commented out for now because if it's left in, it causes the merge sort to mess up and override all the edits (that are not overlapping initially) to the same value (causing overlapping edits) and I'm not sure why.